__tests__/app.tests.js

```diff
@@ -46,3 +46,41 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("/api/articles/:articles_id", () => {
+  test("GET: 200 sends a single article to the client", () => {
+    const articleID = 2;
+    return request(app)
+      .get(`/api/articles/${articleID}`)
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        expect(article).toMatchObject({
+          article_id: articleID,
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+  test("GET:404 sends an appropriate status and error message when given a valid but non-existent id", () => {
+    return request(app)
+      .get("/api/articles/9999")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("article does not exist");
```

Comment:
Nothing wrong with this, I'd recommend some consistency though. In the test above you're destructuring well, then in this test a longer dot notation chain.